src/libraries/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -203,6 +203,7 @@ public static partial class HttpVersion
         public static readonly System.Version Version10;
         public static readonly System.Version Version11;
         public static readonly System.Version Version20;
+        public static readonly System.Version Version30;
```

Comment:
@scalablecory please make sure to have all these new public API documented in the implementation with triple slash comments so that we can automatically port them in the future.